var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -26,12 +30,13 @@ class PyAwkward(PythonPackage):
     patch('pybind11.patch', when="@:1.2.2")
     patch('pybind11_02.patch', when="@1.2.3:")
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools@42.0:', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools@42.0:', type=('build', 'run'))
```
Setuptools is imported at run-time